pkg/etcd/client.go

```diff
@@ -51,6 +51,15 @@ const (
 	etcdRequestProgressDuration = 1 * time.Second
 	// etcdWatchChBufferSize is arbitrarily specified, it will be modified in the future
 	etcdWatchChBufferSize = 16
+	// etcdTxnTimeoutDuration represents the timeout duration for committing a
+	// transaction to Etcd
+	etcdTxnTimeoutDuration = 30 * time.Second
+)
+
+var (
+	TxnEmptyCmps    = []clientv3.Cmp{}
+	TxnEmptyOpsThen = []clientv3.Op{}
+	TxnEmptyOpsElse = []clientv3.Op{}
 )
```

Comment:
We can move these variables to `client_test.go` since they are used in test only.

Reply:
We also use it in etcd_worker and etcd.go, such as
https://github.com/pingcap/tiflow/pull/4474/files#diff-1f744cd3130fdaa4293a04fa689c341aa1086bc64278c7a5af46e7774dc16cd9R389